hazelcast-sql/src/test/java/com/hazelcast/jet/sql/SqlLimitTest.java

```diff
@@ -195,6 +197,25 @@ public void limitWithNegativeDynamicParameter() {
                 .hasMessageContaining("LIMIT value cannot be negative");
     }
 
+    @Test
+    public void limitWithProjection() {
+        final String sql = "SELECT ABS(v) "
```

Comment:
I would use a projection that alters the output, `ABS` actually leaves it _untouched_.

Reply:
Don't think it matters at all in this case though. Optimizer can't get rid of it because we don't have any rules that would simplify calls in case the input is fixed and as long as projection is there, the test verifies exactly what it should - that limit works with projections present.